catalog/index.yaml

```diff
@@ -0,0 +1,243 @@
+          `openshift-sandboxed-containers-operator` namespace, which is automatically
+          created if it does not exist. Attempting to install the OpenShift
+          sandboxed containers Operator in a namespace other than
+          `openshift-sandboxed-containers-operator` causes the installation to fail.
+
+        - For Approval Strategy, ensure that Automatic, which is the default value,
+          is selected. OpenShift sandboxed containers automatically updates when a new
+          z-stream release is available.
+
+        - Click Install to make the Operator available to the OpenShift sandboxed
+          containers namespace.
+
+        - The OpenShift sandboxed containers Operator is now installed on your
+          cluster. You can trigger the Operator by enabling the runtime on your cluster.
+          You can do this by creating a `KataConfig` CustomResourceDefinition(CRD) instance. For this click
+          on "create instance" on the operator overview page.
+
+      # Documentation
+      See the official documentation [here](https://docs.openshift.com/container-platform/latest/sandboxed_containers/understanding-sandboxed-containers.html)
+    displayName: OpenShift sandboxed containers Operator
+    installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+    keywords:
+    - sandboxed-containers
+    - kata
+    labels:
+      operatorframework.io/arch.amd64: supported
+      operatorframework.io/os.linux: supported
+    links:
+    - name: Sandboxed Containers Operator
+      url: https://www.github.com/openshift/sandboxed-containers-operator
+    maintainers:
+    - email: support@redhat.com'
+      name: '''Red Hat'
+    maturity: beta
+    provider:
+      name: Red Hat
+relatedImages:
+- image: gcr.io/kubebuilder/kube-rbac-proxy@sha256:d99a8d144816b951a67648c12c0b988936ccd25cf3754f3cd85ab8c01592248f
+  name: ""
+- image: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator-bundle:v1.6.0
+  name: ""
+- image: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator:v1.6.0
+  name: ""
+- image: registry.redhat.io/openshift-sandboxed-containers/osc-cloud-api-adaptor-rhel9:latest
+  name: caa
+- image: registry.redhat.io/openshift-sandboxed-containers/osc-cloud-api-adaptor-webhook-rhel9:latest
+  name: peerpods-webhook
+- image: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel9:latest
+  name: kata-monitor
+- image: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-builder-rhel9:latest
+  name: podvm-builder
+- image: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-payload-rhel9:latest
+  name: podvm-payload
+schema: olm.bundle
+---
+schema: olm.channel
+package: sandboxed-containers-operator
+name: stable
+entries:
+  - name: sandboxed-containers-operator.v1.6.0
```

Comment:
AFAIK we also need the following because of `olm.skipRange` :

```
    replaces: sandboxed-containers-operator.v1.6.0
```

See [KATA-2894](https://issues.redhat.com/browse/KATA-2894) for an explanation.

Reply:
Adding